rules/dotnet-upgrade-for-complex-projects/rule.md

```diff
@@ -67,19 +69,12 @@ What this will allow you to do is add your target framework and compile the code
 ```csharp
 <TargetFrameworks>net472;net8.0</TargetFrameworks>
 ```
-::: greybox
-Figure: Target to multiple target frameworks.
-:::
-
-![Figure: Multiple TFMs](target-to-multiple-TFMs.png)
 
-::: greybox
-Figure: Git changes for targeting to multiple target frameworks.
-:::
+![Figure: Git changes for targeting to multiple target frameworks](target-to-multiple-TFMs.png)
 
 # Upgrading
 
-At this point, ensure your project can target both the .NET Framework and the new target framework. Iterate through the following steps to gain an understanding of how many projects are ready and how much work lies ahead.
+At this point, ensure your project can target both the .NET Framework and the new target .NET. Some of the projects might not support both platforms right away and you can follow these steps to fix the issues and have a better understanding of how much work it might lies ahead.
```

Comment:
```suggestion
At this point, ensure your project can target both the .NET Framework and the new target .NET. Some of the projects might not support both platforms right away and you can follow these steps to fix the issues and better understand how much work might lie ahead.
```

A little Grammarly fix as "it might lies ahead" doesn't sound right, anymore.